ebssurrogate/scripts/surrogate-bootstrap.sh

```diff
@@ -259,6 +259,9 @@ function clean_system {
 	mkdir /mnt/var/log/wal-g
 	touch /mnt/var/log/wal-g/{backup-push.log,backup-fetch.log,wal-push.log,wal-fetch.log,pitr.log}
 
+	#Creatre Sysstat directory for SAR
+	mkdir /mnt/var/log/sysstat
```

Comment:
Potentially using an if clause first to validate the existence and if not, create. 

Anyway, not a blocker and will approve